### DISCUSSION TITLE
Three hypens open properties in Obsidian 1.4.14

### DISCUSSION THREAD

**Discussion by andresespinosapc** [SebastianMC/obsidian-custom-sort#105] [Q&A]
When I type three hypens Obsidian converts it into a properties block. It seems that because of that the ordering is not being applied.

**Comment by SebastianMC** [comment]
Hi @andresespinosapc

yes, the recent releases of Obsidian introduced a new way of editing the YAML metadata.
You can use the new method and edit YAML metadata via the properties UI or you can stay with the previous plain-text method.

You can find detailed instructions here [How to create or edit a multi-line property in Obsidian](https://github.com/SebastianMC/obsidian-custom-sort/wiki/How-to-create-or-edit-a-multi%E2%80%90line-property-in-Obsidian%3F)

When you type three hyphens in the top of a note it automatically switches to the property editor. If you want to stay within the property editor, you can follow the instructions video 1 or 2.

If you like to use plain text editing, you can switch to 'source mode', as shown in the video 3.

Please let me know if it helps or if you think one more instructional video would be needed
